Makefile

```diff
@@ -15,7 +15,9 @@ endif
 install:
 	@luarocks make OPENSSL_DIR=$(OPENSSL_DIR) CRYPTO_DIR=$(OPENSSL_DIR)
 
-dev: install
+dev:
+	@luarocks remove kong
+	@luarocks make OPENSSL_DIR=$(OPENSSL_DIR) CRYPTO_DIR=$(OPENSSL_DIR)
```

Comment:
I would have suggested something like `dev: uninstall install` and adding an `uninstall` rule with the `@luarocks remove` command, but that assumes that `make` is never called with `-j`. So I think repeating the `@luarocks make` will make do.